src/Sulu/Bundle/SecurityBundle/Controller/UserController.php

```diff
@@ -32,6 +33,8 @@
 use Sulu\Component\Rest\RestHelperInterface;
 use Sulu\Component\Rest\ListBuilder\Doctrine\DoctrineListBuilderFactory;
 use Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor\DoctrineFieldDescriptor;
+use Symfony\Component\Validator\Constraints\Email as EmailConstraint;
```

Comment:
@marcelmoosbrugger could it be that this constraint and validator is never used?